tests/functional/api/BackbeatServer.js

```diff
@@ -88,8 +88,35 @@ function getRequest(path, done) {
     req.end();
 }
 
-describe('Backbeat Server', () => {
-    describe('healthcheck route', () => {
+describe('Backbeat Server', function f() {
```

Comment:
The tests here have only been re-ordered to fit inside their describe details. Other than that, timeouts were added to all blocks

Reply:
Why don't you enforce timeouts using mocha.opts so set the same timeout globally?

Reply:
I'm not sure about adding a global timeout, but I can set it for this file from the package.json script since the script only targets this file